bin/pyenv-win-venv.ps1

```diff
@@ -71,9 +71,21 @@ function  main {
         elseif (test-path -PathType container "$pyenv_versions_dir\$subcommand2") {
             if ($subcommand3 -ne "self") {
                 if (!(test-path -PathType container "$app_env_dir\$subcommand3")) {
-                    Write-Host "Installing env: $subcommand3 using Python v$subcommand2"
                     pyenv shell $subcommand2
-                    python -m venv "$app_env_dir\$subcommand3"
+
+                    # Get active python version, split it to get Major.Patch, conver to float
+                    $pythonVersionString = python -c "import platform;print(platform.python_version())"
+                    $pythonVersionArrray = $pythonVersionString.Split(".")
+                    $pythonVersionStringFloat = $pythonVersionArrray[0] + "." + $pythonVersionArrray[1]
+                    $pythonVersionFloat = [decimal]$pythonVersionStringFloat
```

Comment:
Why are not getting the Python version from $subcommand2 instead of parsing it from Python itself? Is it because of issue #8? I have added a fix for that in [issue8_fix](https://github.com/pyenv-win/pyenv-win-venv/tree/issue8_fix)

Reply:
Note: All the subcommand variables name have been incremented by 1 in the testing branches so $subcommand2 is now $subcommand3.
$subcommand1 is now reserved for "ps1" or "bat" which is used to recognize the shell.

Reply:
Hmm. In that case, it makes sense to do it your way.
Do you want to look at the case of Python v2.7.0 not having pip to install virtualenv? Only v2.7.9+ seems to have Pip installed. I will look into this later tonight but feel free to debug this if you have the free time.

Reply:
Sounds like something we would like to fix before we merge this. How did you install 2.7.0 without pip? When I install it with pyenv-win even 2.5 have pip installed. Which I don't fully understand how, it seems like they run ensure pip but I thought that was new from 3.4.

Reply:
Could be an one off thing. Let me try v2.7 again and verify if pip is installed by pyenv

Reply:
![image](https://user-images.githubusercontent.com/70950611/229071566-7350ebee-9cbc-4544-b186-6ff78ecacaff.png)

Reply:
what does `python -m ensurepip` say?

Reply:
`get-pip` was an option but it seems to fail for any versions below v2.7.0
Issue discussing this: https://github.com/pypa/get-pip/issues/26
![image](https://user-images.githubusercontent.com/70950611/229129892-c70ef392-6c96-4bc0-be03-7809fb8ebe39.png)

Reply:
The current code for your ref
```py
if ([System.Version]"$pythonVersionFloat" -lt [System.Version]"3.3") {
    if ([System.Version]"$pythonVersionFloat" -lt [System.Version]"2.7.9") {
        # Install pip
        Invoke-WebRequest https://bootstrap.pypa.io/pip/2.7/get-pip.py -o "$app_dir\get-pip.py"
        python "$app_dir\get-pip.py" 'pip==20.3.1'
    }

    python -W ignore:DEPRECATION -m pip --disable-pip-version-check install --quiet virtualenv
    python -m virtualenv --quiet "$app_env_dir\$subcommand4"
}
else {
    python -m venv "$app_env_dir\$subcommand4"
}
```

Reply:
I think I have read that people use ex_setup https://bootstrap.pypa.io/ez_setup.py

Reply:
```
Download error on https://pypi.python.org/simple/pip/: [Errno 1] _ssl.c:480: error:140770FC:SSL routines:SSL23_GET_SERVER_HELLO:unknown protocol -- Some packages may not be found!
```

I could not get it to install pip or virtualenv either. I get this SSL error. :(

Reply:
Using get-pip I get this error 


```
DEPRECATION: Python 2.6 is no longer supported by the Python core team, please upgrade your Python. A future version of pip will drop support for Python 2.6
Collecting pip<10
c:\users\jerakin\appdata\local\temp\tmpmfmcrm\pip.zip\pip\_vendor\urllib3\util\ssl_.py:339: SNIMissingWarning: An HTTPS request has been made, but the SNI (Subject Name Indication) extension to TLS is not available on this platform. This may cause the server to present an incorrect TLS certificate, which can cause validation failures. You can upgrade to a newer version of Python to solve this. For more information, see https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
c:\users\jerakin\appdata\local\temp\tmpmfmcrm\pip.zip\pip\_vendor\urllib3\util\ssl_.py:137: InsecurePlatformWarning: A true SSLContext object is not available. This prevents urllib3 from configuring SSL appropriately and may cause certain SSL connections to fail. You can upgrade to a newer version of Python to solve this. For more information, see https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  Retrying (Retry(total=4, connect=None, read=None, redirect=None, status=None)) after connection broken by 'SSLError(SSLError(1, '_ssl.c:490: error:140770FC:SSL routines:SSL23_GET_SERVER_HELLO:unknown protocol'),)': /simple/pip/
c:\users\jerakin\appdata\local\temp\tmpmfmcrm\pip.zip\pip\_vendor\urllib3\util\ssl_.py:137: InsecurePlatformWarning: A true SSLContext object is not available. This prevents urllib3 from configuring SSL appropriately and may cause certain SSL connections to fail. You can upgrade to a newer version of Python to solve this. For more information, see https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  Could not fetch URL https://pypi.python.org/simple/pip/: There was a problem confirming the ssl certificate: HTTPSConnectionPool(host='pypi.python.org', port=443): Max retries exceeded with url: /simple/pip/ (Caused by SSLError(SSLError(1, '_ssl.c:490: error:140770FC:SSL routines:SSL23_GET_SERVER_HELLO:unknown protocol'),)) - skipping
  Could not find a version that satisfies the requirement pip<10 (from versions: )
No matching distribution found for pip<10
```

Having support for Python versions less than 2.7.9 doesn't look plausible. Might not be worth the headache. 2.7 is used frequently but that still (though a lot of industries are trying to phase it out) so having support for the version there we can is probably good?

Reply:
Yea. Adding support for this seems very complicated. We can add support for versions 2.7.9+ since anything is better than nothing. Is your PR tested with 2.7.9+? I can do some testing later tonight and merge the PR if it works fine.